src/Message.php

```diff
@@ -41,16 +58,17 @@ final class Message
     private array $context;
 
     /**
-     * @param mixed $level Log message level.
+     * @param string $level Log message level.
      * @param string|Stringable $message Log message.
-     * @param array $context Log message context.
+     * @param array<string, mixed> $context Log message context.
+     * @psalm-param LogMessageContext $context
      *
      * @throws InvalidArgumentException for invalid log message level.
      *
      * @see LoggerTrait::log()
      * @see LogLevel
      */
-    public function __construct(mixed $level, string|Stringable $message, array $context = [])
+    public function __construct(string $level, string|Stringable $message, array $context = [])
```

Comment:
Need changelog about change type of constructor parameter (seems, this is bug fix)

Reply:
While it's usually a string, the type `mixed` is specified in the PSR:

```php
  /**
     * Logs with an arbitrary level.
     *
     * @param mixed $level
     * @param string $message
     * @param array $context
     * @return void
     */
    public function log($level, $message, array $context = array());
```

https://www.php-fig.org/psr/psr-3/

Reply:
I don't think it's a good idea to change that before it's changed in the PSR.

Reply:
This implementation use strings only. We can set this by type. In `log()` method throws exception for non-string values also, but in methods we don't can change type because interface.

Reply:
OK.